package.json

```diff
@@ -83,7 +83,7 @@
     "uuid": "^3.4.0"
   },
   "peerDependencies": {
-    "@zowe/imperative": "^5.13.1"
+    "@zowe/imperative": "^8.0.0-next"
```

Comment:
should we do a range of `^8.0.0-next.0 <8.0.0` (or `^8.0.0-next.0 <9.0.0` to allow `8.x`)